core/src/components/radio-button/radio-button.stories.js

```diff
@@ -0,0 +1,53 @@
+export default {
+  title: "Components/Radio Button",
+};
+
+export const Default = () => `
+  <label class="fudis-radio-button">
```

Comment:
label for="id" link to input and value property to input

Reply:
Added `value` attribute to all RadioButton stories.